### ISSUE TITLE
How can we use `import` in `background.ts`

### ISSUE THREAD

**Issue by Matt-V50** [Jervis2049/vite-plugin-crx-mv3#14]
When I use `import` to invoke other `api`, I got this error below:

```log
Uncaught SyntaxError: Cannot use import statement outside a module
```

Is there any way to solve this?

https://github.com/Jervis2049/vite-plugin-crx-mv3/blob/e0c27aadfacaa7aa02736589e6f61437562c36a4/examples/crx-vue-drawer/src/background.ts

**Comment by Matt-V50** [comment]
Add `type: module` at the background in the manifest could solve this problem

But maybe we need to make some changes here

```js
    manifest.background = {
      service_worker: SERVICE_WORK_DEV_PATH
    };
```

**Comment by Jervis2049** [comment]
Thank you for the correction, I have fixed the problem in v0.1.1, please try again.

**Comment by Matt-V50** [comment]
> Thank you for the correction, I have fixed the problem in v0.1.1, please try again.

Cool, Fixed
